src/simple_commander/main.py

```diff
@@ -238,13 +240,23 @@ def kill(self, other_unit, units):
         del units[self.id]
 
 
+__game = None
+
+def get_game(height=None, width=None, invaders_count=None, notify_clients=None):
+    global __game
+
+    if not __game and height and width and invaders_count is not None:
```

Comment:
In this implementation with global instance of class there is no reason to leave __new__ method in GameController.